rosbag2_cpp/src/rosbag2_cpp/message_definitions/local_message_definition_source.cpp

```diff
@@ -28,7 +25,11 @@
 #include <unordered_set>
 #include <utility>
 
-namespace rosbag2_storage_mcap::internal
+#include <ament_index_cpp/get_package_share_directory.hpp>
+#include <ament_index_cpp/get_resource.hpp>
+#include <ament_index_cpp/get_resources.hpp>
```

Comment:
Small hint from CLion code inspection. The last two includes seems redundant.
```suggestion
#include <ament_index_cpp/get_package_share_directory.hpp>
```